File: OneCodeTeam/How to clear session when the web browser is closed in ASP.NET/[VB]-How to clear session when the web browser is closed in ASP.NET/VB/VBASPNETClearSessiononBrowserClose/SessionInfo.vb
Public Class SessionInfo
    ' Dummy session variable value
    Private _sessionValue As String

    ' Closed time of the browser
    Private _closedTime As DateTime

    Public Property SessionValue() As String
        Get
            Return _sessionValue
        End Get
        Set(value As String)
            _sessionValue = value
        End Set
    End Property


    Public Property BrowserClosedTime() As DateTime
        Get
            Return _closedTime
        End Get
        Set(value As DateTime)
            _closedTime = value
        End Set
    End Property

End Class

File: OneCodeTeam/How to clear session when the web browser is closed in ASP.NET/[C#]-How to clear session when the web browser is closed in ASP.NET/C#/CSASPNETClearSession/SessionInfo.cs
using System;
using System.Collections.Generic;
using System.Web;
using System.Xml.Linq;
using System.Linq;

namespace CSASPNETClearSession
{    
    public class SessionInfo
    {
        // Dummy session variable value
        private string _sessionValue;

        // Closed time of the browser
        private DateTime _closedTime;

        public string SessionValue
        {
            get
            {
                return _sessionValue;
            }
            set
            {
                _sessionValue = value;
            }
        }
    

        public DateTime BrowserClosedTime
        {
            get
            {
                return _closedTime;
            }
            set
            {
                _closedTime = value;
            }
        }
  
    }
}
